styles.css

```diff
@@ -90,29 +101,47 @@ main {
 	max-width: 2000px;
 	height: auto;
 	position: relative;
+	padding: 20px;
 }
 
 header img {
 	width: 100%;
 }
 
 .portfolio {
+	display: flex;
+	flex-flow: column nowrap;
+	border: 1px solid;
+	background-color:var(--dark-blue); 
+	color: var(--my-white);
+	margin: 20px;
 	padding: 20px;
 }
 
+.portfolio-card-container{
+	display: grid;
```

Comment:
you might find flex more 'flexible' here, you can share the space using flex-grow: 50% (I think, there is a flex command that allows you to do this) and then allow a wrap